### ISSUE TITLE
SDXL checkpoints are drawing blank heatmaps

### ISSUE THREAD

**Issue by Chadius** [rockerBOO/sd-webui-daam#13]
I'm on Windows 11
Stable-Diffusion 1.7
NVidia 4070 TI with CUDA
daam-demo works as expected
All extensions except this one are disabled (I reinstalled it about 15 minutes ago)

When I try the example image of a cute cat wearing sunglasses at the beach, it only works for 1.5 checkpoints. My SDXL checkpoints draw a blank white square.

```
DAAM: Trace attention heatmaps for prompt: 01<00:00, 12.78it/s]
DAAM:   cute cat wearing sunglasses at the beach
DAAM: Attention words:
DAAM:   cute
DAAM:   cat
DAAM:   sunglasses
DAAM:   beach
100%|##########| 20/20 [00:02<00:00,  7.93it/s]
unknown call trace_each_layers#| 20/20 [00:02<00:00,  9.51it/s]
Total progress: 100%|##########| 20/20 [00:02<00:00,  7.14it/s]
Total progress: 100%|##########| 20/20 [00:02<00:00,  9.51it/s]
```

This is my webui-user.bat . Maybe I have an incompatible package?
```
@echo off

set PYTHON=
set GIT=
set VENV_DIR=
:: set COMMANDLINE_ARGS=--update-check --opt-sdp-attention --no-half-vae --xformers  --medvram
set COMMANDLINE_ARGS=--update-check --opt-sdp-attention --no-half-vae --xformers
set TORCH_COMMAND=pip install torchvision torch==2.1.2
set XFORMERS_PACKAGE=xformers==0.0.23 --index-url https://download.pytorch.org/whl/cu118

call webui.bat
```

**Comment by rockerBOO** [comment]
Yes I have been working to fix up 1.7 but have been having issues identifying what has changed. I am getting the white heatmaps for SDXL on 1.7 as well so I will work to get it all resolved sometime this weekend. Thank you for reporting this.

**Comment by Chadius** [comment]
Thanks for taking a look at this!

**Comment by rockerBOO** [comment]
Sorry for the delay, I spend some time trying to resolve this multiple times since we spoke but still haven't gotten resolution. Will probably need to work through it more thoroughly but haven't had the time recently. If desired this extension should work properly on 1.6 version of automatic1111.

I will try to resolve this as best as I can when I can. Thank you.

**Comment by Chadius** [comment]
Thanks for the update, and trying to fix this. Please take your time.
